server/test_route.py

```diff
@@ -38,5 +38,75 @@ def test_fetch_emotion(client):
 # TODO: sentiment testing
 # def test_fetch_sentiment(client):
 
-# TODO: summary testing
-# def test_fetch_summary(client):
+# Medium length input (400 words) - tests the basic API call works, and we get a response that 'summarises', i.e. shorter than the input.
```

Comment:
Can we rename this to be "default" not "medium" to keep it consistent to how its defined in the frontend and the processing. 🙏

Reply:
The `medium` here just refers to the length of the actual input, not the requested `summary_len_option` (if that makes sense), I think technically they are different things. We are using `SummaryLengthOption.DEFAULT` for the requested summary for all these tests